frontend/mr-admin-flate/src/components/tabell/AvtaleTabell.tsx

```diff
@@ -11,12 +10,18 @@ import { AVTALE_PAGE_SIZE } from "../../constants";
 import { capitalizeEveryWord, formaterDato } from "../../utils/Utils";
 import { Avtalestatus } from "../statuselementer/Avtalestatus";
 import Lenke from "mulighetsrommet-veileder-flate/src/components/lenke/Lenke";
+import { SorteringAvtaler } from "mulighetsrommet-api-client";
+import { useState } from "react";
+import { Sortering } from "./Utils";
 
 export const AvtaleTabell = () => {
   const { data, isLoading, isError } = useAvtaler();
+  const [filter, setFilter] = useAtom(avtaleFilter);
   const [page, setPage] = useAtom(avtalePaginationAtom);
-  const [sort, setSort] = useState();
-  const rowsPerPage = 15;
+  const [sort, setSort] = useState<Sortering>({
+    orderBy: "navn",
+    direction: "ascending",
+  });
```

Comment:
Vi kan dra denne ut i en egen hook som kan gjenbrukes for de andre tabellene. Vi kan gjerne se på det sammen.

Reply:
Har laget Trello-kort på det!

Reply:
Du kan gjøre det i denne PRen, trenger ikke eget kort på det